app/views/admin/shared/_vis_header.html.erb

```diff
@@ -9,11 +9,12 @@
               <input type="text" />
               <input type="submit" />
           </li></form>
+          <li><a class="rounded button add_new_layer" href="#/add-new-layer">Browse</a></li>
```

Comment:
My take was that the button was in the wireframes but got missed, and the wireframes I looked at on #44 and #45 showed the button between the search bar and "Annotate" marked as "Browse". Is there a more appropriate place to look?